langchain/src/prompts/base.ts

```diff
@@ -35,14 +39,37 @@ export abstract class BasePromptTemplate implements BasePromptTemplateInput {
 
   outputParser?: BaseOutputParser;
 
+  partialVariables?: InputValues;
+
   constructor(input: BasePromptTemplateInput) {
-    const { inputVariables } = input;
+    const { inputVariables, partialVariables } = input;
     if (inputVariables.includes("stop")) {
       throw new Error(
         "Cannot have an input variable named 'stop', as it is used internally, please rename."
       );
     }
     Object.assign(this, input);
+    Object.assign(this, partialVariables);
```

Comment:
This doesn't do what you intent I think. This will copy each key of `partialVariables` onto `this` but then on the next method it looks like you access this.partialVariables. A simple assignment will do here